packages/api/src/services/repository/mappers/repository-list.mapper.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {RepositoryListMapper} from './repository-list.mapper';
+import {Repository, RepositoryList} from '../../../models/repositories';
+
+describe('RepositoryListMapper', () => {
+
+  let repositoryListMapper: RepositoryListMapper;
+
+  beforeEach(() => {
+    repositoryListMapper = new RepositoryListMapper();
+  });
+
+  test('Should return an instance of RepositoryList with repositories', () => {
+    const repoOneId = 'repo-one-id';
+    const repoTwoId = 'repo-two-id';
+    const repoThreeId = 'repo-three-id';
+    const repoThreeLocation = 'http://localhost:7002';
+    const rawRepositoryObjects = {
+      '': [{
+        id: repoOneId
+      }, {
+        id: repoTwoId
+      }],
+      'http://localhost:7002': [
+        {
+          id: repoThreeId,
+          location: repoThreeLocation
+        }
+      ]
+    };
+
+    const repositoryList = repositoryListMapper.mapToModel(rawRepositoryObjects);
+
+    expect(repositoryList).toBeInstanceOf(RepositoryList);
+    const repositories = repositoryList.getItems();
+    expect(repositories).toHaveLength(3);
+    expect(repositories).toContainEqual(new Repository({id: repoOneId} as Repository));
+    expect(repositories).toContainEqual(new Repository({id: repoTwoId} as Repository));
```

Comment:
Eventually, check if the two instances above doesn't have a location

Reply:
Updated raw data and added a check for locations.